port/cli/models.go

```diff
@@ -43,26 +43,37 @@ type (
 		Required   []string                     `json:"required,omitempty"`
 	}
 
+	ActionInvocationMethod struct {
+		Type string `json:"type,omitempty"`
+		Url  string `json:"url,omitempty"`
+	}
+
+	BlueprintChangelogDesintaion struct {
```

Comment:
We can name it `ChangelogDestination` since it already is a field of `Blueprint`